### PR TITLE
Generate recommendations for all users at once.

### PR DESCRIPTION
**Pull request by vansika**
# Problem
It took around 16 hours to generate recommendations for around 1700 users. We have to generate recommendations for each user separately since the candidate set for each user is different but the operations which are performed on each set of recommendations can definitely be done for all users at once. I think it will reduce time to some extent.

# Solution
Get mbids corresponding to recording_ids for all users at once.
Scale the ratings for all users at once.

***EDIT***
I have lately realised that recommendations can be generated for all users at once. So in this script, we are performing all operations for all users at once. I am expecting runtime to fall greatly.

### CODE REVIEW

**Review by mayhem — APPROVED** [review]
as usual hard to follow, but as far as I can tell, it looks ok!

**Comment by vansika** [comment]
@mayhem can we merge, release and test?
